sysbuild/Kconfig.suit

```diff
@@ -6,9 +6,9 @@
 
 menu "SUIT"
 
-# 'default y if SOC_SERIES_NRF54HX' to be added when SUIT is to be enabled globally.
 menuconfig SUIT_ENVELOPE
 	bool "Create SUIT envelope"
+	default y if SOC_SERIES_NRF54HX
```

Comment:
@e-rk will this influence flash_companion? Shouldn't the flash_companion app set `SUIT_ENVELOPE` to `n`?

Reply:
No, this is the sysbuild Kconfig that must be enabled always with or without the flash companion.